src/main/webapp/js/instructorSearch.js

```diff
@@ -8,3 +8,11 @@ $(function() {
 	
 	$("div[id^=plainCommentText]").css("margin-left","15px");
 });
+
```

Comment:
Can you remove the changes unrelated to this issue? So that in future when we look at this changes for this issue, we will not see unrelated code.